src/main/java/duke/helper/TaskList.java

```diff
@@ -0,0 +1,78 @@
+package duke.helper;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * TaskList class that handles all the tasks
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+    private Ui ui;
+
+    /**
+     * Constructor of the TaskList class
+     *
+     * @param tasks tasks to be stored
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+        this.ui = new Ui();
+    }
+
+    /**
+     * Adds the task to the task list
+     *
+     * @param task task to be added
+     */
+    public void addToTasks(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Prints all the tasks stored in the list
+     */
+    public String outputList() {
+        String output = "Here are the tasks in your list:\n";
+        for (int i = 0; i < tasks.size(); i++) {
+            output += (i + 1) + "." + tasks.get(i) + "\n";
+        }
+        return output;
+    }
+
+    /**
+     * Deletes the task from the tasklist
+     *
+     * @param taskNo Task number to be deleted
+     * @return task that has been deleted
+     */
+    public Task deleteTask(int taskNo) {
+        Task task = tasks.get(taskNo);
```

Comment:
Is it better to add a try catch block because there might be possible indexoutofbound error